cmd/storage.go

```diff
@@ -44,7 +44,9 @@ var storageCreateCmd = &cobra.Command{
 		checkError(err, "")
 		projectName := project.GetCurrent(client)
 		componentName := getComponent(client, storageComponent, applicationName, projectName)
-
+		//validate storage path
+		err = validateStoragePath(client, storagePath, componentName, applicationName)
+		checkError(err, "")
 		_, err = storage.Create(client, args[0], storageSize, storagePath, componentName, applicationName)
```

Comment:
I feel as though we should start organizing the code a bit better and add newlines on seperate sections. For instance... The above code talks about validating storage path, while the code below here is regarding Create. Perhaps adding a newline between this code would make things look cleaner.

Reply:
thanks @cdrage